person.rb

```diff
@@ -1,9 +1,19 @@
-class Person
-  def initialize(name: 'Unknown', age: nil, parent_permission: true)
+require_relative 'nameable'
+require_relative 'Decorator'
+require_relative 'Trimmer_Decorator'
+require_relative 'Capitalize_Decorator'
```

Comment:
- [ ] **[OPTIONAL]** These imports are wrong too, however, as they are imported for testing purposes (logging the results into the console) I suggest that once you test the code, you remove them to keep your class files clean at all times. 🤓 🔧